compat/fsmonitor/fsm-settings-win32.c

```diff
@@ -108,7 +108,7 @@ static enum fsmonitor_reason is_remote(struct repository *r)
 
 	driveType = GetDriveTypeW(wfullpath);
 	trace_printf_key(&trace_fsmonitor,
-			 "DriveType '%s' L'%S' (%u)",
+			 "DriveType '%s' L'%ls' (%u)",
```

Comment:
I can go either way on this.  This is WIN32-only code and %S is allowed there.
However, it it makes the pedantic warnings go away I guess it is worth it.
I probably need to do a sweep of whole series for these in the Win32 code.

Reply:
I'll fix this one and a few other instanced in compat/fsmonitor/fsm-*-win32.c code in my builtin-fsmonitor-part4 series.